prime-angular-frontend/src/app/lib/modules/ngx-busy/busy.config.ts

```diff
@@ -8,5 +8,6 @@ export const busyConfig = {
   delay: 50,
   minDuration: 600,
   disableAnimation: true,
-  wrapperClass: 'ng-busy'
+  wrapperClass: 'ng-busy',
+  message: 'Wait patiently, the submission can take up to a minute.'
```

Comment:
If the message omitted, the default message would be 'Please wait...'.  from my understanding we do not want to use the word "Please". Also, the message  (and/or spinner) will never show up until we explicitly set their flags. The other way to work around this is to set the default message to an empty string and then pass a busy message whenever we want, not difficult, just adds an extra step for each submission.

Reply:
So, I added the message here, because if we omit that then the default message is "Please wait...", from my understanding, during refinement it was mentioned not to use the word "Please". I could set the default message to an empty string and then we'd pass the message we want but I though why not just add the message in the config since we never use the loader message and if we do we can easily override it. To work around it, we can set the config message to an empty string.